internal/shim-sev/src/syscall.rs

```diff
@@ -537,7 +537,9 @@ impl Handler {
 
     pub fn clock_gettime(&self) -> Result<usize, libc::c_int> {
         let clk_id = self.a;
-        let trusted = self.b as *mut libc::timespec;
+
+        // FIXME: check `trusted`, if in payload space
```

Comment:
Please file an issue for this; we should absolutely be verifying userspace pointers. Once it's filed, put a link to the issue here.

Reply:
done

Reply:
https://github.com/enarx/enarx-keepldr/issues/78